partner_area/views/res_partner.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_res_partner_form_inherit" model="ir.ui.view">
+        <field name="name">view.res.partner.form.inherit</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//notebook/page[1]" position="attributes">
+                <attribute name="autofocus"/>
+            </xpath>
+            <xpath expr="//notebook/page[1]" position="before">
+                <page string="Area" autofocus="autofocus">
+                    <field name="shape_line_ids" mode="map">
+                        <map library="drawing" string="Shape">
```

Comment:
The view `map` has been changed to `google_map`. On the enterprise, Odoo has this view type